submissions/JuliaNovak/frogger-game/app.js

```diff
@@ -0,0 +1,118 @@
+// Enemies our player must avoid
+var Enemy = function (x, y) {
+   this.x = x;
+   this.y = y;
+   // Variables applied to each of our instances go here,
+   // we've provided one for you to get started
+
+   // The image/sprite for our enemies, this uses
+   // a helper we've provided to easily load images
+   this.sprite = "images/enemy-bug.png";
+};
+
+// Update the enemy's position, required method for game
+// Parameter: dt, a time delta between ticks
+Enemy.prototype.update = function (dt) {
+   const fieldWidth = 505;
+   // You should multiply any movement by the dt parameter
+   // which will ensure the game runs at the same speed for
+   // all computers.
+   for (let i = 0; i < 200; i++) {
+      this.x += dt;
+      if (this.x > fieldWidth) {
+         this.x = Math.floor(Math.random() * -200);
+      }
+   }
+};
+
+// Draw the enemy on the screen, required method for game
+Enemy.prototype.render = function () {
+   ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+};
+
+// Now instantiate your Enemy.
+// Place all enemy objects in an array called allEnemies
+const allEnemies = [new Enemy(0, 63), new Enemy(-30, 145), new Enemy(-50, 230)];
```

Comment:
Mystery numbers. Check the requirements with regards to constants.
Also make this fragment DRY.